simulator/mips/mips_instr.h

```diff
@@ -326,6 +330,8 @@ class MIPSInstr
 
         bool get_writes_dst() const { return writes_dst; }
 
+        bool is_accumulating_instr() const { return accumulating_instr; }
```

Comment:
You need to add `bool is_accumulating_instr() const { return false; }` to RISCVInstr